examples/05-RNN-examples.ipynb

```diff
@@ -37,7 +37,15 @@
    "metadata": {
```

Comment:
<p>I guess this title could be changed (there's no longer output_chunk_length)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/354/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Ah yes, thanks, I missed that. Done.